docs/documentation/angular-cli.md

```diff
@@ -0,0 +1,88 @@
+<!-- Links in /docs/documentation should NOT have `.md` at the end, because they end up in our wiki at release. -->
+
+# Angular CLI Config Schema
+
+## Options
+
+- **project**: The global configuration of the project.
+  - *name* (`string`): The name of the project.
+  - *ejected*(`boolean`): Whether or not this project was ejected.
+
+
+- **apps** (`array`): Properties of the different applications in this project.
+  - *name* (`string`): Name of the app.
+  - *root* (`string`): The root directory of the app.
+  - *outDir* (`string`): The output directory for build results. Default is `dist/`.
+  - *assets* (`array`): List of application assets.
+  - *deployUrl* (`string`): ...
+  - *index* (`string`): The name of the start HTML file. Default is `index.hrml`
+  - *main* (`string`): The name of the main entry-point file.
+  - *polyfills* (`string`): The name of the main entry-point file.
+  - *test* (`string`): The name of the test entry-point file.
+  - *tsconfig* (`string`): The name of the TypeScript configuration file. Default is `tsconfig.app.json`.
+  - *testTsconfig* (`string`): The name of the TypeScript configuration file for unit tests.
+  - *prefix* (`string`): The prefix to apply to generated selectors.
+  - *serviceWorker* (`boolean`): Experimental support for a service worker from @angular/service-worker. Default is false.
```

Comment:
Everywhere there's a literal value, please make it code (using back ticks). `false` and `true` are not following this actually.